tests/sparsezoo/utils/test_data.py

```diff
@@ -11,204 +11,115 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Iterable
 
-import numpy
-import numpy as np
+
+from collections import OrderedDict
+
 import pytest
 
-from sparsezoo.utils import Dataset
+from sparsezoo.utils.data import DataLoader, RandomDataset
+
+
+# TESTS for DataLoader
 
 
 @pytest.fixture
-def dummy_dataset():
-    return Dataset(data=[np.random.rand(100, 10)], name="dummy-dataset")
+def dataset():
```

Comment:
Can we add return types to the fixture method definitions?

Reply:
Done!
![](https://media2.giphy.com/media/26u4lOMA8JKSnL9Uk/giphy.gif)